app.js

```diff
@@ -18,7 +20,10 @@ let key;
 const keyPath = `${appPath}/key.txt`;
 try { key = fs.readFileSync(keyPath) } catch (_) {}
 
-const archive = drive.createArchive(key, { live: true });
+const archive = drive.createArchive(key, {
+  live: true,
+  file: name => raf(`${appPath}/files/${name}`)
```

Comment:
+1 ~/Downloads folder for Mac. not sure about x-platform. I think `home-dir` is the module for getting the users home dir. I think the folder should be easy to change eventually but for now Downloads seems like a sensible default